docker/runtime/java/function/src/main/java/io/kubeless/Example.java

```diff
@@ -0,0 +1,7 @@
+package io.kubeless;
+
+public class Example {
```

Comment:
probably noob question but can we avoid having a `class` or is it mandatory for some reason?

Reply:
Java ia very verbose. Unavoidable.